pythonbits/bb.py

```diff
@@ -425,14 +446,25 @@ def _render_resolution(self):
     def _render_additional(self):
         additional = []
         audio_tracks = self['tracks']['audio']
+        file_path = self['path']
         text_tracks = self['tracks']['text']
 
+        # assumes the user:
+        # 1) has a folder structure that contains 'remux' in the path
+        #    (ex. /mnt/movies/remux/filename.ext)
+        # 2) the file is properly named, containing 'remux'
+        if any(x in file_path for x in ('remux', 'Remux', 'REMUX')):
+            if 'REMUX' not in additional:
+                additional.append('REMUX')
+
```

Comment:
If you want to add remux, please just check if "Remux" is in self['guess']['other'].